### DISCUSSION TITLE
Looking for yaml files(Documentation) that get deployed when you deploy a service through linkerd cli, or more objects.

### DISCUSSION THREAD

**Discussion by chowmean** [linkerd/linkerd2#5568] [General]
Need to know about the kubernetes objects that are created when we deploy objects through linkerd cli. Any link will be really helpful.

**Comment by olix0r** [comment]
@chowmean `linkerd install` generates yaml. You can simply save the output with, for instance `linkerd install >linkerd.yaml`. Is this what you're looking for?
